### PR TITLE
Added GLB import information and new asset tasks (Draco + FBX folder)

### PR DESCRIPTION
**Pull request by yaustar**
Fixes part of the assets section for https://github.com/playcanvas/developer.playcanvas.com/issues/465

TLDR:

- Adds missing asset tasks for Draco Mesh Compression and Create FBX Folder
- glTF GLB is now the preferred format over FBX

### CODE REVIEW

File: content/en/user-manual/assets/models/building.md [hunk 1]
```diff
@@ -38,7 +59,7 @@ Regardless which modeling application you are using, there a number of things to
 
 [Blender][4] is a fantastic free tool for 3D modeling and it is perfect for generating 3D assets for PlayCanvas.
 
-Blender supports exporting to both COLLADA and FBX, but the built-in exporters do have some limitations.
+Blender supports exporting to GLB, FBX and COLLADA but the built-in exporters do have some limitations.
 
 ### **Map Types**
```
Comment (yaustar) [inline content/en/user-manual/assets/models/building.md:65]: I don't know if the rest of the page is still accurate so left as is

File: content/en/user-manual/assets/import-pipeline/import-hierarchy.md [hunk 2]
```diff
@@ -20,7 +24,9 @@ Scroll down to 'Asset Tasks' and enable 'Import Hierarchy':
 
 ## Importing models
 
-Drag the model file into the 'Assets Panel':
+A full tutorial on importing your first model and animation can be found [here][first_model_animation_import].
+
+As an overview, when you drag the model file into the 'Assets Panel':
```
Comment (willeastcott) [inline content/en/user-manual/assets/import-pipeline/import-hierarchy.md:29]: Add newline between para and image.

File: content/en/user-manual/assets/import-pipeline/index.md [hunk 3]
```diff
@@ -10,13 +10,11 @@ Some assets don't need to be imported before they can be used. For example a PNG
 
 ## Asset Tasks
 
-When a source asset is uploaded that needs to be imported. PlayCanvas starts an Asset Task to perform this import process on our server. You can see the asset tasks that you are currently running in the Asset Panel.
-
-![asset tasks][1]
+When a source asset is uploaded that needs to be imported. PlayCanvas starts an Asset Task to perform this import process on our server.
```
Comment (willeastcott) [inline content/en/user-manual/assets/import-pipeline/index.md:13]:
```suggestion
When a source asset is uploaded, PlayCanvas starts an Asset Task to perform this import process on our server.
```

File: content/en/user-manual/assets/import-pipeline/index.md [hunk 4]
```diff
@@ -10,13 +10,11 @@ Some assets don't need to be imported before they can be used. For example a PNG
...
 
 There are a variety of options available to tune the behavior of the import pipeline to suit your needs.
 
-<img loading="lazy" src="/images/user-manual/assets/import-pipeline/asset-tasks.png" width="360px">
+<img loading="lazy" src="/images/user-manual/assets/import-pipeline/asset-tasks.png" width="480px">
```
Comment (willeastcott) [inline content/en/user-manual/assets/import-pipeline/index.md:17]: No `px` on `width` value.

File: content/en/user-manual/assets/import-pipeline/index.md [hunk 5]
```diff
@@ -75,11 +73,24 @@ Enabled by default on new projects, imported models and animations will create G
 
 Only available if using [Convert to GLB](#convert-to-glb) option. When a model file is imported, a template asset is created that contains the full hierarchy of the model as entities allowing to you to manipulate them directly in the Editor. See more information about this feature [here][3].
 
+### Mesh Compression
+
+Only available if using [Convert to GLB](#convert-to-glb) option. Setting this to a compression format will automatically compress mesh data when importing or re-importing model files. This can drastically reduce the size of GLB files at the cost of some runtime decompression cost.
+
+If using Draco compression, remember to import the Draco WASM module into the project otherwise the models will not load.
+
+<img loading="lazy" src="/images/user-manual/assets/import-pipeline/draco-import-button.png" width="480px">
```
Comment (willeastcott) [inline content/en/user-manual/assets/import-pipeline/index.md:82]: No `px` on width attribute.

File: content/en/user-manual/assets/models/building.md [hunk 6]
```diff
@@ -6,7 +6,28 @@ position: 2
...
+The import of GLBs supports the following glTF features for incredibly fast scene creation:
+
+- Node hierarchy
+- Standard material
+- Ratified material extensions
+- Animated skeletons and skinning data
+- Morph targets
+- Cameras
+- Draco compressed models
+- Light punctual support
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/building.md:20]:
```suggestion
- Punctual light support
```

File: content/en/user-manual/assets/models/building.md [hunk 7]
```diff
@@ -6,7 +6,28 @@ position: 2
...
+
+As such, to target PlayCanvas, you do not have to use any special art tools plugins and there are no PlayCanvas-specific workflows you must follow.
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/building.md:22]:
```suggestion
As such, to target PlayCanvas, you do not have to use any special art tool plugins and there are no PlayCanvas-specific workflows you must follow.
```

File: content/en/user-manual/assets/models/building.md [hunk 8]
```diff
@@ -6,7 +6,28 @@ position: 2
...
+
+![GLB Import Example][14]
+
+(Model by Loïc Norgeot and mosquito scan by Geoffrey Marchal for [Sketchfab][13] licensed under [CC BY 4.0][12]).
+
+PlayCanvas also supports importing of FBX mode which is another common interchange format. However, it doesn't retain material properties and you will need to configure them after import.
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/building.md:28]:
```suggestion
PlayCanvas also supports the importing of FBX files which is another common interchange format. However, FBX materials are not imported as faithfully as glTF materials and might require some editing after import.
```

File: content/en/user-manual/assets/models/exporting.md [hunk 9]
```diff
@@ -4,11 +4,29 @@ layout: usermanual-page.hbs
 position: 3
 ---
 
-PlayCanvas can import 3D content in the following formats: FBX, OBJ, 3DS and COLLADA (DAE). We strongly recommend you use FBX since this the most robust and well tested interchange format.
+PlayCanvas can import 3D content in the following formats: glTF binary (GLB), FBX, OBJ, 3DS and COLLADA (DAE). We strongly recommend you use GLB as it's an open source industry standard and fantastically supported in the Editor.
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/exporting.md:7]:
```suggestion
PlayCanvas can import 3D content in the following formats: glTF binary (GLB), FBX, OBJ, 3DS and COLLADA (DAE). We strongly recommend you use GLB as it's an open source industry standard and very well supported in the Editor.
```

File: content/en/user-manual/assets/models/exporting.md [hunk 10]
```diff
@@ -4,11 +4,29 @@ layout: usermanual-page.hbs
...
-Here are some tips for exporting to the FBX file format:
+![GLB Import Example][glb-import-gif]
+
+(Model by Loïc Norgeot and mosquito scan by Geoffrey Marchal for [Sketchfab][mosquito] licensed under [CC BY 4.0][cc-40]).
+
+If GLB is not available, then please use FBX as it is a robust and well tested interchange industry format.
+
+## General export tips
+
+* Only export what you need from your scene. For example, if your scene contains a red car and a blue car but you only want to import the red car, export only the red car. Your modelling application should have an 'Export Selected' or similar option.
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/exporting.md:17]:
```suggestion
* Only export what you need from your scene. For example, if your scene contains a red car and a blue car but you only want to import the red car, export only the red car. Your modeling application should have an 'Export Selected' or similar option.
```

Comment (willeastcott) [reply content/en/user-manual/assets/models/exporting.md:17]: Modelling is UK English, modeling is US English.

File: content/en/user-manual/assets/models/exporting.md [hunk 11]
```diff
@@ -4,11 +4,29 @@ layout: usermanual-page.hbs
...
+
+## Export to GLB file format tips
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/exporting.md:19]:
```suggestion
## Tips for Exporting to GLB
```

File: content/en/user-manual/assets/models/exporting.md [hunk 12]
```diff
@@ -4,11 +4,29 @@ layout: usermanual-page.hbs
...
+
+* Set the format to glTF Binary (GLB).
+* Ensure that materials and images are set to be exported and embedded in the GLB. Otherwise, you must upload all scene textures separately and relink all textures to materials and re-configure the materials parameters.
+
+## Export to FBX file format tips
```
Comment (willeastcott) [inline content/en/user-manual/assets/models/exporting.md:24]:
```suggestion
## Tips for Exporting to FBX
```

**Review by willeastcott — APPROVED** [review]
Approving with a bunch of comments/suggestions.

**Pull request merged**